test-case/multiple-pipeline.sh

```diff
@@ -34,9 +34,9 @@
 set -e
 
 # shellcheck source=case-lib/lib.sh
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
-OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg path, default to environment variable: TPLG'
```

Comment:
I will add this common string change into my task list. Will ping you for overall clean up.

Reply:
I will keep this change in this PR for the multiple-pipeline.sh, because leave only one shellcheck warning makes me feel uncomfortable. also, I will update `tplg` in the description to `topology`.

Reply:
I have barely tested it but this seems to work better and without warning:
```shell
OPT_DESC['t']="tplg file, default value is env TPLG: $TPLG"
```
If the above is not possible for some unknown reason, here's a quoting trick to avoid the warning:
```shell
OPT_DESC['t']='tplg file, default value is env TPLG: $''TPLG'
```